packages/components/src/internal/renderers/ProjectColumnRenderer.tsx

```diff
@@ -0,0 +1,13 @@
+import { Map } from 'immutable';
+import React, { PureComponent, ReactNode } from 'react';
+
+interface Props {
+    data: Map<any, any>;
+}
+
+export class ProjectColumnRenderer extends PureComponent<Props> {
+    render(): ReactNode {
+        const { data } = this.props;
+        return data?.get('displayValue');
```

Comment:
Do we plan to resolve those url in the future?

Reply:
Currently, there are no plans to resolve these as links.